tests/support_server/windows/win2019_config.pm

```diff
@@ -0,0 +1,51 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Install VDMP tools to enable network drivers correctly.
+#
+# Maintainer: mmartins <mmartins@suse.com>
+
+#use base 'basetest' ;
+use base 'windowsbasetest';
+use warnings;
+use testapi;
+
+sub run {
+    my $self = shift;
+
+    #first login
+    assert_screen "windows-installed-ok";
+    send_key "ctrl-alt-delete";
+    assert_screen "windows_server_login", timeout => 60;
+    type_string "N0tS3cr3t@";
+    send_key "ret";
+    assert_screen "wint_manage_server", timeout => 150;
```

Comment:
This is the same as in `tests/support_server/windows/win2019_boot.pm` so I'd create a function in `lib/windowsbasetest.pm` and call it on both places.

Reply:
Good sugestion. Implemented.